### PR TITLE
Add note to LandingSite - BigJumpMidairShinespark

### PR DESCRIPTION
**Pull request by dragonsinth**
https://videos.maprando.com/video/382

I could not make this trick work with a max height jump, even without high jump boots.

### CODE REVIEW

File: region/crateria/central/Landing Site.json [hunk 1]
```diff
@@ -1075,6 +1075,7 @@
       "clearsObstacles": ["A"],
       "note": [
         "Store the spark on the right side of the ledge.  Run left and do a big jump towards the gauntlet door and then midair spark at the right time.",
+        "Do not jump to max height or you will likely run out of shinecharge frames, instead short the jump a little.",
```
Comment (kjbranch) [inline region/crateria/central/Landing Site.json:1078]:
```suggestion
        "Do not jump to max height or Samus will run out of shinecharge frames, instead short the jump a little.",
```

File: region/crateria/central/Landing Site.json [hunk 2]
```diff
@@ -1113,6 +1114,7 @@
       ],
       "note": [
         "Store the spark on the right side of the ledge.  Run left and do a big jump towards the gauntlet door and then midair spark at the right time.",
+        "Do not jump to max height or you will likely run out of shinecharge frames, instead short the jump a little.",
```
Comment (kjbranch) [inline region/crateria/central/Landing Site.json:1117]:
```suggestion
        "Do not jump to max height or Samus will run out of shinecharge frames, instead short the jump a little.",
```

**Review by kjbranch — APPROVED** [review]

**Comment by dragonsinth** [comment]
@kjbranch thank you, good suggestions

**Comment by blkerby** [comment]
I did some testing and found it's possible to do with a full height jump but it requires jumping somewhat early and the timing feels tighter. And with a shorter jump you don't have as much fall speed which makes it easier to position vertically for the spark. So shorting the jump seems like good advice

**Pull request merged**